### PR TITLE
Add Deep Anchor Links

### PR DESCRIPTION
**Pull request by sahilakos**
I had merged your enterprise branch to our master at first by mistake and then added other stuff on top. I just removed the call to partial _enterprise in pricing page for now, so it looks the same. Anchor links were added using Anchor js.

### CODE REVIEW

File: assets/js/main.js [hunk 1]
```diff
@@ -353,4 +361,13 @@ $(function () { // This prevents global vars
...
   });
+});
+
+/* This adds hover anchor links to all headers */
+$(function() {
+  anchors.options = {
+    placement: 'left'
+  };
+  anchors.add();
+  anchors.add('.terms .terms-row strong');
```
Comment (josh-padnick) [inline assets/js/main.js]: Can we be a little more targeted about applying this style? `terms` is a generic enough word that I could imagine it being used in another context and getting a surprise anchor applied to it. Perhaps you can give each page's `<body>` an `id` and then specifically declare the `id`s as part of the CSS selector here.

File: pages/pricing/_enterprise.html [hunk 2]
```diff
@@ -0,0 +1,9 @@
+<div id="pricing-faqs" class="row row-page-header">
+  <div class="col-xs-12">
+    <h2 style="padding-top: 27px;">Enterprise Agreements</h2>
+    <p>Need to customize our <a href="/terms">Terms of Service</a>? Download our terms as a <a href="#">Microsoft Word
+      File</a>, but please note that we require an Enterprise Service Plan for all custom contracts. The Enterprise
+      Service Plan includes a faster Service Level Agreement, more user licenses, and other benefits.</p>
+    <p>For more information, <a href="/contact"> Contact Us</a>.</p>
+  </div>
+</div>
```
Comment (josh-padnick) [inline pages/pricing/_enterprise.html]: Should this be removed from this PR?

File: pages/pricing/_pricing-faqs.html [hunk 3]
```diff
@@ -1,6 +1,6 @@
 <div id="pricing-faqs" class="row row-page-header">
   <div class="col-xs-12">
-    <h2 class="text-center">Pricing FAQ's</h2>
+    <h2>Pricing FAQ's</h2>
```
Comment (josh-padnick) [inline pages/pricing/_pricing-faqs.html]: I think this actually looks better centered for now.

File: pages/pricing/_enterprise.html (hunk 2 above)
Comment (sahilakos) [reply pages/pricing/_enterprise.html]: It's not being called in the page build. I think we should be fine. I can remove it if you'd like.

File: pages/pricing/_pricing-faqs.html (hunk 3 above)
Comment (sahilakos) [reply pages/pricing/_pricing-faqs.html]: Yes

File: pages/pricing/_enterprise.html (hunk 2 above)
Comment (brikis98) [reply pages/pricing/_enterprise.html]: Probably best to remove to keep the PRs clean and self-contained.

**Comment by josh-padnick** [comment]
It doesn't look like I have push permissions to this fork for some reason, so I'm going to merge to our master and manually delete the offending enterprise section so we can deploy these changes live.

FYI, I wanted to do this in prep for our Houston legal website changes.

@sahilakos Could you grant @brikis98 and I "write" privileges on this repo?

**Comment by josh-padnick** [comment]
@sahilakos I was just doing some quality control before I published these changes, and I realized that the problem is that now a handful links have the anchor icon appearing that shouldn't have it:

image

image

image

image

image

I didn't originally intend for the links to be added to every header on the site, but it actually winds up being a handy reference and I don't find it distracting. So can you just make the above fixes in a new PR?

**Comment by sahilakos** [comment]
@josh-padnick Just making sure, you want me to remove the anchor links from the headers in specific images you attached?

**Comment by josh-padnick** [comment]
@sahilakos Yes, that's correct.

**Pull request merged**